plugins/inventory/ovirt.py

```diff
@@ -9,6 +10,7 @@
     name: ovirt
     plugin_type: inventory
     short_description: oVirt inventory source
+    version_added: "2.10"
```

Comment:
Because of the collection, there is different versioning. 
Please replace with: `version_added: "1.0.0"`